src/big-query-functions/big-query-service-class.ts

```diff
@@ -0,0 +1,58 @@
+import { Issue } from "../api-functions/get-issues-by-project";
+import { BigQuery } from "@google-cloud/bigquery";
+
+export default class BQ {
+    //Creates a new table in the dataset
+    static async createTable(  
+        datasetId: string, // Existing dataset
+        tableId: string, // Table to be created
+        schema: any
+    ) : Promise<void> {
+        const bigquery = new BigQuery();
```

Comment:
Have this as a class property. so that you can use it all other methods. dont create this instance inside all the methods.